lib/ppx_mysql_runtime.ml

```diff
@@ -1,14 +1,10 @@
 (********************************************************************************)
 (** {1 Public functions and values}                                             *)
+
 (********************************************************************************)
 
 let identity x = x
+let map_option f = function Some x -> Some (f x) | None -> None
+let get_option = function Some x -> x | None -> assert false
 
-let map_option f = function
-    | Some x -> Some (f x)
-    | None   -> None
-
-let get_option = function
-    | Some x -> x
-    | None   -> assert false (* FIXME *)
-
+(* FIXME *)
```

Comment:
It kinda screwed with this comment. Completely out of place now.

Reply:
Yeah, but also I absolutely hate in-line comments. I would like a language which made this a syntax error.